build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
but if it is required to be build.sh for the multi-stage support then ignore this comment :-)